### PR TITLE
RELATED: BB-1526 Remove duplicated executionResult request

### PR DESCRIPTION
**Pull request by martinmosko**
---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [x] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [x] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [x] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [x] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [x] Successful `extended test - examples`
- [x] Successful `extended test - storybook`
- [x] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-app-components:
- gdc-analytical-designer:
- gdc-dashboards:

# Related Jira tasks

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3875/ : FAILURE in 8m 15s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4907/ : SUCCESS in 47s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3876/ : FAILURE in 23m 04s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4908/ : SUCCESS in 15s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3878/ : SUCCESS in 14m 53s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4910/ : SUCCESS in 9s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3880/ : FAILURE in 5m 27s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4913/ : SUCCESS in 52s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3883/ : FAILURE in 4m 31s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4917/ : SUCCESS in 12s

File: src/components/core/pivotTable/tests/RowLoadingElement.spec.tsx [hunk 1]
```diff
@@ -1,8 +1,8 @@
 // (C) 2019 GoodData Corporation
 import { shallow } from "enzyme";
 import * as React from "react";
-import { LoadingComponent } from "../../..";
-import { RowLoadingElement } from "./RowLoadingElement";
+import { LoadingComponent } from "../../../../index";
```
Comment (yanlipnican) [inline src/components/core/pivotTable/tests/RowLoadingElement.spec.tsx]: fix pls

File: src/components/core/pivotTable/agGridDataSourceUtils.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+interface ICachedPageRequest {
+    requestParams: {
+        resultSpec: AFM.IResultSpec;
+        limit: number[];
+        offset: number[];
+    };
+    response: Execution.IExecutionResponses | null;
+}
+
+export const wrapGetPageWithCaching = (getPage: IGetPage): IGetPage => {
```
Comment (yanlipnican) [inline src/components/core/pivotTable/agGridDataSourceUtils.ts]: async pls

Comment (martinmosko) [reply src/components/core/pivotTable/agGridDataSourceUtils.ts]: done

File: src/components/core/pivotTable/tests/RowLoadingElement.spec.tsx (hunk 1 above)
Comment (martinmosko) [reply src/components/core/pivotTable/tests/RowLoadingElement.spec.tsx]: done

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by martinmosko** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3885/ : SUCCESS in 10m 32s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4919/ : SUCCESS in 16s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2121/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1471/ : SUCCESS in 13m 16s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4920/ : SUCCESS in 10s

**Comment by martinmosko** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1019.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1472/ : SUCCESS in 31m 50s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4921/ : SUCCESS in 35s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2123/ : SUCCESS in 10m 00s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3890/ : SUCCESS in 6m 13s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1024/ : SUCCESS in 16s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/1179/ : FAILURE in 21m 31s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3908/ : SUCCESS in 16m 12s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4948/ : SUCCESS in 12s

**Comment by yanlipnican** [comment]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2138/ : SUCCESS in 10m 16s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/3914/ : SUCCESS in 14m 50s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1033/ : SUCCESS in 24s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/1192/ : SUCCESS in 25m 41s

**Pull request merged**